src/test/shell/bazel/bazel_symlink_test.sh

```diff
@@ -88,21 +104,31 @@ EOF
 }
 
 function test_smoke() {
+  if "$is_windows"; then
+    warn "Skipping test on Windows: Unresolved symlinks are not supported yet"
```

Comment:
Is there an existing github issue for this, that we can reference in a TODO (here and at other occurrences)?

Reply:
Sure, let's replace the warning with a

    // TODO(#10298): Support unresolved symlinks on Windows.

comment. Warnings are logspam anyway -- the fact that we're not running on windows is relevant for maintainers of the test suite, but not people viewing the test logs.

Reply:
Ok, done